homeassistant/components/device_tracker/__init__.py

```diff
@@ -486,7 +486,7 @@ def get_vendor_for_mac(self):
             return None
 
         # prevent lookup of invalid macs
-        if not len(self.mac.split(':')) == 6:
+        if not len(self.mac) == 17:
```

Comment:
@vmulpuru out of curiosity, what is the reason the fact a device is discovered through bluetooth or BLE is stored in the MAC address as a prefix and not as an attribute to the entity? 

Referring to: https://github.com/home-assistant/home-assistant/commit/c1ca13d61350712871d0b5e4ca065a4724c9291e